City-of-Helsinki/berth-reservations-admin
# VEN-622, VEN-731 | Fetch order information from BE for BillsCard and BillingHistoryCard

## Description :sparkles:

After this patch billing information is fetched from the backend in the customer page.

### Closes :no_good_woman:

[VEN-622](https://helsinkisolutionoffice.atlassian.net/browse/VEN-622)
[VEN-731](https://helsinkisolutionoffice.atlassian.net/browse/VEN-731)

## Testing :alembic:

### Automated tests :gear:️

- Updated `BillsCard.test.jsx`.
- Added `BillingHistoryCard.test.tsx`

### Manual testing :construction_worker_man:

- Open `/customers/UHJvZmlsZU5vZGU6MmI1YzViMmItM2FjMi00MTAzLWI2ODAtZjIxZWJmODBlNDMy`
- Open bill information should be displayed as provided by the backend.
- Billing history should be displayed as provided by the backend.

## Additional notes :spiral_notepad:

The "Open bill" button is still a placeholder, will need to be implemented once the page/modal it leads to is done.
